### PR TITLE
Features/#91 hh building apartment integration

### PR DESCRIPTION
**Pull request by IlkaCu**
As described in #91 this PR includes the download and import of household, building and apartment data into our workflow.

### CODE REVIEW

**Comment by IlkaCu** [comment]
Meta data for the zensus tables are not included yet. I will tackle this in a separate issue.

File: src/egon/data/importing/zensus/__init__.py [hunk 1]
```diff
@@ -24,46 +24,104 @@ def download_zensus_pop():
...
+    zensus_url = zensus_config["source"]["url"]
+    zensus_path = zensus_misc_processed["path_table_map"].keys()
+    url_path_map = list(zip(zensus_url, zensus_path))
+
+    for url, path in url_path_map:
+        target_file_misc = os.path.join(os.path.dirname(__file__), path)
+
+        if not os.path.isfile(target_file_misc):
+            urlretrieve(url, target_file_misc)
+        url_path_map
```
Comment (ClaraBuettner) [inline src/egon/data/importing/zensus/__init__.py:46]: line 46 could be deleted, right?

**Comment by ClaraBuettner** [comment]
I tested the branch and everything worked. It just took a while, especially task 'import-zensus-misc' which needed about 2 hours. Was that similar for you?

**Comment by IlkaCu** [comment]
Yes, importing the data to the local database takes a while. This data set is one of those which in theory need to be imported just once.

File: src/egon/data/importing/zensus/__init__.py (hunk 1 above)
Comment (IlkaCu) [reply src/egon/data/importing/zensus/__init__.py:46]: Right!

**Review by ClaraBuettner — APPROVED** [review]

**Pull request merged**